### DISCUSSION TITLE
Gist previews not displaying correctly

### DISCUSSION THREAD

**Discussion by Elliria** [community/community#53460] [Other Feature Feedback, Questions, & Ideas]
### Bug

I've come across what seems to be a bug in the Gist previews or thumbnails or pins or whatever you refer to them as (what you see when you're on your Gist profile page). It seems that if you have a nested list, at least the first level of the sub-list is not indented in the preview/thumbnail/pin. Note that I haven't tested this with several nested levels, but you'll surely want to. It should be indented in both places. Bug?

### See it on GitHub:
You can see this in action this way:
1. Visit [my GitHub profile page](https://github.com/Elliria).
2. Look at the contents of the **Clipboard.md** Gist's preview/thumbnail/pin.
3. Note that its last visible line (the `• This contains selected data.` line) is not indented.
4. Click on the preview/thumbnail/pin to open [the Gist that it links to](https://gist.github.com/Elliria/6925e9e4c92597019aa4eebc86c5a5f7).
5. Note that that line (the `• This contains selected data.` line) is indented, as I intended it to be.

### Or see the screenshots:
Here's a screenshot that shows the Gist preview/thumbnail/pin/whatever:

![GistPreviewThumbnailPin](https://user-images.githubusercontent.com/25792925/233741404-3b70f3b9-3df5-48b6-869e-96ec06929bd8.png)

And here's a screenshot that shows the Gist:

![Gist](https://user-images.githubusercontent.com/25792925/233741420-00a3aae7-b7b0-4bbd-ad38-0058e2f6841d.png)

**Comment by asheroto** [comment]
Looks like previews aren't working at all on gists now, at least not on markdown files.

**Comment by Elliria** [comment]
You're right. It's now showing the raw code of the document:
![ChangedPinnedPreview](https://github.com/community/community/assets/25792925/f5b6d31d-10d9-48d5-9a10-2f6dd99f371e)

**Comment by asheroto** [comment]
You're looks even better than mine...

**Edit:**
![image](https://github.com/community/community/assets/49938263/2cf6a200-f783-4cbb-89c2-5e95034cc016)

**Preview:**
![image](https://github.com/community/community/assets/49938263/11f59e64-4220-47dd-b293-d79f3104ecca)

**Comment by Elliria** [comment]
Ah, I see. My mistake. In the steps above, I should have said to visit [my GitHub profile page](https://github.com/Elliria). I've corrected the information above now.

Your screenshots are of the Gist editor. What I meant to refer to in this discussion/issue is the previews in the Gist pins that you can see when you're on your GitHub profile page (the pins on [your GitHub profile page](https://github.com/asheroto)).

But out of curiosity, I checked what my document looks like in the Gist editor as a comparison to yours. They're definitely different and I'm not sure why. Here are mine:

![1](https://github.com/community/community/assets/25792925/683a2a15-b36a-495a-984d-c63cf1bfbb35)
![2](https://github.com/community/community/assets/25792925/d774e990-3807-4035-8b3a-4d55da49a1bd)

Some of your syntax may not be what GitHub is expecting. I've sometimes found that if, for example, I don't put a blank line between two types of elements in a Markdown file, it won't display correctly or if I don't put the proper number of spaces before indented list items, they don't display properly. It could be something like that.

I just visited your Gists and looked at how some of them are previewing and you've got several `.md` files that are previewing properly. I couldn't find the `README.md` file from your example to take a look at the source code, though.

**Comment by asheroto** [comment]
Ah, sorry, I was skimming your Issue and didn't catch the info about the pins.

Here is the link to the Gist:
https://gist.github.com/asheroto/cfa26dd00177a03c81635ea774406b2b

I reported the issue to GitHub support. Here's a video:


https://github.com/community/community/assets/49938263/ba4ac9a0-c959-48dd-9677-90c801fc09ae

**Comment by Elliria** [comment]
That's really strange. I went to [your Gist](https://gist.github.com/asheroto/cfa26dd00177a03c81635ea774406b2b), grabbed the raw source of the [README.md](https://gist.github.com/asheroto/cfa26dd00177a03c81635ea774406b2b#file-readme-md) file, and put it into a new Gist on my account and my preview of it looks as I would expect it to:
![asheroto](https://github.com/community/community/assets/25792925/676f129f-e275-4675-8d55-7218cb859755)

**Comment by asheroto** [comment]
That's so strange. Thanks for testing on your end. I guess I'll let you know what GitHub support says. I mentioned this Issue so maybe they will chime in here!

**Reply by Elliria** [reply]
Yeah, good luck with it, and let me know if they need me to do further testing.

**Reply by asheroto** [reply]
They were able to replicate the issue and are having the engineers check it out.

**Reply by Elliria** [reply]
Oh, that's really good news. Now let's hope they take a look at this one next.

**Reply by asheroto** [reply]
FYI they said it's on their to do list to take a closer look at.

**Reply by Elliria** [reply]
That's more really good news. Thanks.
